Source/BestProjectEver/Private/PlayerController/BPE_PlayerController.cpp

```diff
@@ -39,6 +42,28 @@ void ABPE_PlayerController::OnPossess(APawn* InPawn)
 	}
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+void ABPE_PlayerController::SetPlayerName()
+{
+	const ABPE_GameState* GameStateReference = Cast<ABPE_GameState>(GetWorld()->GetGameState());
+	if(IsValid(GameStateReference) && IsValid(PlayerState))
+	{
+		PlayerState->SetPlayerName(FString("Player ") + FString::FromInt(GameStateReference->PlayerArray.IndexOfByKey(PlayerState) + 1));
+	}
+}
+
+//----------------------------------------------------------------------------------------------------------------------
+void ABPE_PlayerController::Server_TravelToMap_Implementation(const FString& FURL)
```

Comment:
Why is the variable called FURL?

Reply:
![image](https://user-images.githubusercontent.com/85327912/175394490-bd8321f8-155b-4a54-82a2-d74dc8fb14ea.png)

Reply:
![image](https://user-images.githubusercontent.com/85327912/175394630-47610c8e-a1ba-481e-9338-f9a29da72918.png)